tests/30rooms/05aliases.pl

```diff
@@ -101,3 +101,53 @@
             ->SyTest::pass_on_done( "m.room.canonical_alias rejects missing aliases" );
       });
    };
+
+test "Creators can delete alias",
+   requires => [ $creator_fixture, $room_fixture,
+                 qw( can_create_room_alias )],
+
+   bug => "SYN-83",
+
+   do => sub {
+      my ( $user, $room_id ) = @_;
+      my $server_name = $user->http->server_name;
+      $room_alias = "#Kuke9aic:$server_name";
```

Comment:
Rather than this hardcoded value in the test, a more robust way would be to use a `room_alias_fixture()`; see 10apidoc/30room-create.pl